README.md

```diff
@@ -293,17 +293,12 @@ $ git stash pop
 ```
 
 <a href="i-want-to-discard-my-local-uncommitted-changes"></a>
-### I want to discard my local, uncommitted changes
+### I want to discard my local uncommitted changes(staged and unstaged)
```

Comment:
Add a space between changes and the parens

Reply:
This is done